src/discussions/in-context-topics/data/__factories__/inContextTopics.factory.js

```diff
@@ -0,0 +1,61 @@
+import { Factory } from 'rosie';
+
+import { getApiBaseUrl } from '../../../../data/constants';
+
+Factory.define('topic')
+  .sequence('id', ['topicPrefix'], (idx, topicPrefix) => `${topicPrefix}-${idx}`)
+  .sequence('enabled-in-context', ['enabledInContext'], (idx, enabledInContext) => enabledInContext)
+  .sequence('name', ['topicNamePrefix'], (idx, topicNamePrefix) => `${topicNamePrefix}-${idx}`)
+  .sequence('usage-key', ['usageKey'], (idx, usageKey) => usageKey)
+  .sequence('courseware', ['courseware'], (idx, courseware) => courseware)
+  .attr('thread_counts', () => {
+    Factory.reset('thread-counts');
+    return Factory.build('thread-counts', null, { discussionCount: 1, questionCount: 1 });
+  });
```

Comment:
allow this attribute to get dynamic params for discussion count and question count.